pom.xml

```diff
@@ -44,8 +44,8 @@ THE SOFTWARE.
   <properties>
     <jenkins.version>1.625.3</jenkins.version>
     <java.level>7</java.level>
-    <mavenInterceptorsVersion>1.11</mavenInterceptorsVersion>
-    <mavenVersion>3.1.0</mavenVersion>
+    <mavenInterceptorsVersion>1.12-SNAPSHOT</mavenInterceptorsVersion>
```

Comment:
I agree but it depends how many days the PR will leave. SNAPSHOTs are quickly removed and thus it makes the build KO each time a new SNAPSHOT is published ...